### PR TITLE
7.0 add recreate base file

### PR DESCRIPTION
**Pull request by yvaucher**

### CODE REVIEW

File: base_delivery_carrier_files/wizard/generate_carrier_files_view.xml [hunk 1]
```diff
@@ -17,11 +17,14 @@
...
-                    <separator colspan="4"/>
-                    <group col="3" colspan="4">
-                        <button special="cancel" colspan="1" string="Cancel" icon='gtk-cancel'/>
-                        <button name="action_generate" string="Generate Files" type="object" icon="gtk-execute"/>
+                    <group>
+                      <field name="recreate"/>
                     </group>
+                    <footer>
+                      <button name="action_generate" string="Generate Files" type="object" icon="gtk-execute" class="oe_highlight"/>
+                      or
```
Comment (pedrobaeza) [inline base_delivery_carrier_files/wizard/generate_carrier_files_view.xml:25]: Please include this word in pot file.

File: base_delivery_carrier_files/wizard/generate_carrier_files.py [hunk 2]
```diff
@@ -39,6 +39,11 @@ def _get_picking_ids(self, cr, uid, context=None):
     _columns = {
         'picking_ids': fields.many2many('stock.picking.out',
                                         string='Delivery Orders'),
+        'recreate': fields.boolean(
+            'Recreate files',
+            help="If this option is used, new files will be generated "
+                 "for selected picking even if they already had one.\n"
+                 "By default, pickings hwith exsting file are skipped."),
```
Comment (guewen) [inline base_delivery_carrier_files/wizard/generate_carrier_files.py]: Double typo. Also, prefer "delivery order" over picking.

Comment (yvaucher) [inline base_delivery_carrier_files/wizard/generate_carrier_files.py]: ok good catch 
hwith and exsting

File: base_delivery_carrier_files/wizard/generate_carrier_files_view.xml (hunk 1 above)
Comment (yvaucher) [inline base_delivery_carrier_files/wizard/generate_carrier_files_view.xml:25]: Thanks for the review.

Translations updated

File: base_delivery_carrier_files/stock.py [hunk 3]
```diff
@@ -33,7 +33,8 @@ class stock_picking(orm.Model):
                                                  "has been generated."),
     }
 
-    def generate_carrier_files(self, cr, uid, ids, auto=True, context=None):
+    def generate_carrier_files(self, cr, uid, ids, auto=True,
+                               recreate=False , context=None):
```
Comment (bwrsandman) [inline base_delivery_carrier_files/stock.py]: E203 whitespace before ','

**Comment by bwrsandman** [comment]
:+1:

**Comment by guewen** [comment]
:+1:

**Comment by pedrobaeza** [comment]
:+1:

**Pull request merged**